colossus-metrics/src/main/scala/colossus/metrics/MetricClock.scala

```diff
@@ -42,16 +42,16 @@ class IntervalAggregator(namespace: MetricAddress, interval: FiniteDuration, sna
       context.system.scheduler.scheduleOnce(interval, self, SendTick)
     }
 
-    case Tock(m, v) => {
-      if (v == latestTick) {
-        if(collectors.contains(sender())) {
-          build = build <+> m
+    case Tock(metrics, tick) => {
+      if (tick == latestTick) {
+        if(collectors.contains(sender)) {
+          build = build <+> metrics
           incrementCollected()
-        }else {
-          log.warning(s"Received metrics from an unregistered EventCollector: ${sender()}")
+        } else {
+          log.warning(s"Ignoring metrics from an unregistered EventCollector sent by $sender")
         }
-      }else{
-        log.warning(s"Currently processing tick# $latestTick.  Received a tock message for an outdated tick#: $v.  Ignoring")
+      } else {
+        log.warning(s"Ignoring tock from $sender with tick #$tick as metric clock is at tick #$latestTick")
```

Comment:
Obviously I'm in favor of the new way. At the very least, the existing message has weird punctuation. I also don't see how it is "Currently processing tick". It is processing a tock message and "currently" is redundant language. Where would you suggest I put $sender in the existing log msg?